sphinx-doc/deprecated.rst

```diff
@@ -4,6 +4,20 @@
 Deprecated
 ==========
 
+Features deprecated
+
+v3.x
+____
+
+.. list-table::
+
+   * - Feature
```

Comment:
Is there a different way to typeset this so that it looks more linear? It took me a minute to realize what was going on here.

Reply:
There are other restructured text table formats that read more clearly in the source text, but are significantly more cumbersome to edit:  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#tables